packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -109,14 +106,45 @@ export const switchfield: SwitchFieldTokens = {
   track: {
     backgroundColor: { value: '{colors.background.tertiary.value}' },
     borderRadius: { value: '{radii.xxxl.value}' },
-    checked: {
-      backgroundColor: { value: '{colors.brand.primary.60.value}' },
```

Comment:
Don't we need to leave this for backwards compatibility?

Reply:
Updated to revert these breaking token name changes.